types/lodash/v3/index.d.ts

```diff
@@ -13329,50 +13330,41 @@ declare module _ {
          * @param defaultValue The value returned if the resolved value is undefined.
          * @return Returns the resolved value.
          */
-        get<TObject, TResult>(
-            object: TObject,
-            path: StringRepresentable|StringRepresentable[],
-            defaultValue?: TResult
-        ): TResult;
-
-        /**
-         * @see _.get
-         */
-        get<TResult>(
+        get<TExpected, TDefault = undefined>(
```

Comment:
I don't like that `TExpected` is only used in the return value - this goes against the [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes). See my comment in #19433 for a better way to fix this.

Reply:
hah, nice catch @aj-r, now I see)